languages/csharp/exercises/concept/regular-expressions/.meta/Example.cs

```diff
@@ -0,0 +1,80 @@
+using System.Text.RegularExpressions;
+
+public class LogParser
+{
+    public bool IsMatch(string text)
+    {
+        const string searchArg = @"\[TRC\] | \[DBG\] | \[INF\] | \[ERR\] | \[WRN\] | \[FTL\]";
+        return Regex.Match(text, searchArg, RegexOptions.IgnorePatternWhitespace).Success;
+    }
+
+    public string[] SplitLogLine(string text)
+    {
+        return Regex.Split(text, "<[*^=-]*>");
+    }
+
+    public bool[] AreQuotedPasswords(string[] lines)
+    {
+        bool[] results = new bool[lines.Length];
+        var regex = new Regex(@"^.*""[^\\""]*password[^\\""]*"".*$", RegexOptions.IgnoreCase);
+        int ctr = 0;
+        for (int i = 0; i < lines.Length; i++)
+        {
+            results[i] = regex.IsMatch(lines[i]);
+        }
+
+        return results;
```

Comment:
I was wondering if maybe we can have this be about using `Regex.Matches`, and then have the student return the number of quoted passwords. This would mean changing the `string[]` to a `string` and using the `MultiLine` option, which is one of the more useful options I feel.